packages/backend/src/openapi/webhooks.yaml

```
@@ -0,0 +1,275 @@
+openapi: 3.1.0
+info:
+  title: Rafiki Webhooks
+  version: 1.0.0
+webhooks:
+  incomingPaymentCompleted:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an incoming payment was completed and funds should now be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/incomingPayment'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  incomingPaymentExpired:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an incoming payment has expired and funds should now be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/incomingPayment'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  outgoingPaymentCreated:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an outgoing payment was created and should now be funded.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/outgoingPayment'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  outgoingPaymentCompleted:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an outgoing payment completed. Remaining funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/outgoingPayment'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  outgoingPaymentFailed:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an outgoing payment failed and won't be retried. Funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/outgoingPayment'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  webMonetization:
+    post:
+      requestBody:
+        description: Notify account servicing entity that a Web Monetization payment was received and funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/webMonetization'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+
+components:
+  schemas:
+    incomingPayment:
```

Should this be 
```suggestion
    incomingPaymentEvent:
```
instead? Since the underlying `data` is the `IncomingPayment` itself and the `id` is actually the `eventId`?